src/containers/banner/component.tsx

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from 'react';
+
+import { useT } from '@transifex/react';
+
+import { AnimatePresence, motion } from 'framer-motion';
+
+import Button from 'components/button';
+
+import styles from './styles.module.scss';
+import cx from 'classnames';
+
+import MegaphoneIcon from 'icons/megaphone.svg?react';
+import ExternalLinkIcon from 'icons/external_link.svg?react';
+import CloseIcon from 'icons/close.svg?react';
+import {  FEATURED } from 'router';
+
+const DISMISS_HE_BANNER = 'HE_VIEW_STORY_BANNER';
```

Comment:
Check with Andreia, but I think we don't need to dismiss this banner. It will be always shown on the Half Earth day